db/migrate/20190628084704_create_consultations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateConsultations < ActiveRecord::Migration[6.0]
+  def change
+    create_table :consultations do |t|
+      t.string :title
```

Comment:
I think we can add null false validation here as well.

Reply:
done in #19